### PR TITLE
feat(dropdown): expose show, hide & toggle handlers on default slot

### PR DESCRIPTION
**Pull request by fullstackfool**

### CODE REVIEW

File: src/components/dropdown/UIDropdown.vue [hunk 1]
```diff
@@ -16,7 +16,7 @@
              :style="dropdownStyle"
              role="group"
              @click.stop>
-            <slot />
+            <slot :toggle="toggle" :show="show" :hide="hide" />
```
Comment (nandi95) [inline src/components/dropdown/UIDropdown.vue:19]: Reckon lets also set the expose attribute for the sake of consistency

File: src/components/dropdown/Demo.vue [hunk 2]
```diff
@@ -11,13 +11,18 @@
...
         </template>
 
-        <div class="divide-y">
-            <div v-for="index in range(10)"
-                 :key="index"
-                 class="px-4 py-2 text-color transition-colors select-none hover:bg-gray-700">
-                custom content and stuff plus things {{ index }}
+        <template #default="{hide}">
+            <div class="divide-y">
+                <div class="px-4 py-2 text-color transition-colors select-none hover:bg-gray-700" @click="hide">
```
Comment (nandi95) [inline src/components/dropdown/Demo.vue:16]: nice

File: package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "@karnama/vueish",
-    "version": "0.12.0",
+    "version": "0.12.1",
```
Comment (nandi95) [inline package.json:3]: this is a feature

**Review by nandi95 — APPROVED** [review]
🦦

File: src/components/dropdown/UIDropdown.vue (hunk 1 above)
Comment (fullstackfool) [reply src/components/dropdown/UIDropdown.vue:19]: 👍

**Pull request merged**